src/calcium/CalciumTypes.jl

```diff
@@ -59,11 +59,36 @@ end
 #
 ################################################################################
 
+qqbar_field_doc = raw"""
+    QQBarField <: Field
+    QQBarFieldElem <: FieldElem
+
+The algebraic closure $\overline{\mathbb Q}$ of the rationals $\mathbb Q$ and
+its elements. This is a singleton type, meaning that there is a unique global
+instance of this field, which can be obtained via `algebraic_closure(QQ)`.
+
+```jldoctest
+julia> K = algebraic_closure(QQ)
+Field of algebraic numbers
+
+julia> sqrt(K(2))
```

Comment:
```suggestion
julia> QQBar = algebraic_closure(QQ)
Field of algebraic numbers

julia> sqrt(QQBar(2))
```
this would be one workaround to not use the global object `QQBar` in docstrings